packages/components/src/internal/renderers.spec.tsx

```diff
@@ -292,7 +292,7 @@ describe('HeaderCellDropdown', () => {
 
     test('isSortAsc via view sort', () => {
         const sortObj = { fieldKey: 'column', dir: '+' };
-        const view = ViewInfo.create({ sort: [sortObj] });
+        const view = new ViewInfo({ name: '', sort: [sortObj] });
```

Comment:
was the `name: ''` addition needed? or maybe this was to add a new test case?

Reply:
I forgot to revert this. At one point this was erroring because it was missing a ton of required properties on ViewInfoJson, but I made them all optional because as far as I can tell they are all treated as optional (they all have default values in the default object, or we compute them in the constructor).